scio-examples/src/main/scala/com/spotify/scio/examples/extra/ParquetExample.scala

```diff
@@ -136,9 +137,15 @@ object ParquetExample {
       .saveAsTextFile(args("output"))
   }
 
-  private def typedIn(sc: ScioContext, args: Args): ClosedTap[String] =
-    sc.typedParquetFile[AccountProjection](args("input"))
+  private def typedIn(sc: ScioContext, args: Args): ClosedTap[String] = {
+    // typed API does not support macro generated expressions, so it requires using FilterApi
+    val predicateExp = FilterApi.and(
+      FilterApi.notEq(FilterApi.intColumn("id"), java.lang.Integer.valueOf(0)),
+      FilterApi.lt(FilterApi.intColumn("id"), java.lang.Integer.valueOf(10000))
+    )
+    sc.typedParquetFile[AccountProjection](args("input"), predicate = predicateExp)
```

Comment:
can we have this in another PR as this is not related ?

Reply:
Sure!